backend/src/routes/api/namespaces/namespaceUtils.ts

```diff
@@ -73,14 +73,14 @@ export const applyNamespaceChange = async (
       labels = {
         'opendatahub.io/dashboard': 'true',
       };
+      annotations = {
```

Comment:
Great, this is interesting, it might have happened when merging both the [f/ossm](https://github.com/opendatahub-io/odh-dashboard/tree/f/ossm) and [f/model-serving](https://github.com/opendatahub-io/odh-dashboard/tree/f/model-serving) into incubation, good catch.

This is a weird situation, but I think solving this in `incubation` is the right approach, we need to be careful once we promote ossm to main again @andrewballantyne 


@bartoszmajsak just fix the linting issue and we can merge it.

Reply:
The diff / git annotations aren't helping find which commit brought this. It might be because of the complex nature of how you deal with changes now/your branching strategy. I agree it could have been an auto-merge that went wrong. I managed to pin it to this merge commit: https://github.com/opendatahub-io/odh-dashboard/commit/39956df0dc734b1ba9f5f9b0472d19cc2f54683e#diff-a319cea33b543d7642d6cdd32ffea3771e0fdafb54fe1cfedd4a7d3ebc8930d0

As for the linter - it was a bit confusing at first, but it's fixed in ed8e8943 :)